src/main/java/com/revature/controllers/AccountController.java

```diff
@@ -16,4 +20,40 @@ public class AccountController {
     public AccountController(AccountService accountService) {
         this.accountService = accountService;
     }
+
+    @GetMapping
+    public ResponseEntity<List<Account>> getAllAccounts() {
+        return ResponseEntity.ok().body(accountService.getAllAccounts());
+    }
+
+    @GetMapping("/id/{accountId}")
+    public ResponseEntity<Object> getAccountByAccountId(@PathVariable("accountId") int accountId) {
+        try {
+            return ResponseEntity.accepted().body(accountService.getAccountByAccountId(accountId));
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return ResponseEntity.badRequest().body(e.getMessage());
+        }
+    }
+
+    @GetMapping("/user/{userId}")
+    public ResponseEntity<Object> getAccountByUserId(@PathVariable("userId") int userId) {
+        try {
+            return ResponseEntity.accepted().body(accountService.getAccountByUserId(userId));
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return ResponseEntity.badRequest().body(e.getMessage());
+        }
+    }
+
+    @PostMapping("/new/{userId}")
+    public ResponseEntity<Object> insertLoan(@RequestBody Account account, @PathVariable("userId") int userId) {
```

Comment:
I believe this method should be called `insertAccount` not `insertLoan`

Reply:
Good catch, I was taking some of your code to change into the method for the account section and I totally didn't catch that.